knowledge-base/kb-pdfviewer-viewport-snapshot.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Creating a ViewPort snapshot in RadPdfViewer
+description: Create a snapshot of the visible part of a PDF document in the RadPdfViewer.
```

Comment:
Create a snapshot of the visible part of a PDF document in RadPdfViewer for WPF.